ci/cloudbuild/dockerfiles/fedora-37-bazel.Dockerfile

```diff
@@ -51,3 +51,12 @@ ENV PATH=${CLOUD_SDK_LOCATION}/bin:${PATH}
 RUN curl -o /usr/bin/bazelisk -sSL "https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-${ARCH}" && \
     chmod +x /usr/bin/bazelisk && \
     ln -s /usr/bin/bazelisk /usr/bin/bazel
+
+# Download the packages needed to run Bigtable conformance tests.
+WORKDIR /var/tmp/downloads
+RUN wget -O go.tgz https://go.dev/dl/go1.20.5.linux-amd64.tar.gz
+RUN tar -C /usr/local/ -xzf go.tgz
+ENV GO_LOCATION=/usr/local/go
+ENV PATH=${GO_LOCATION}/bin:${PATH}
+RUN go version
+RUN git clone -b v0.0.1 --single-branch https://github.com/googleapis/cloud-bigtable-clients-test.git
```

Comment:
nit: I doubt `renovate-bot` will recognize this pattern.

Can we use the following:

```Dockerfile
WORKDIR /var/tmp/downloads/cloud-bigtable-clients-test
RUN curl -fsSL https://github.com/googleapis/cloud-bigtable-clients-test/archive/v0.0.1.tar.gz | \
    tar -xzf - --strip-components=1
```

Reply:
Done.